com.bestaticpy.bestatic.yml

```diff
@@ -0,0 +1,16 @@
+id: com.bestaticpy.bestatic
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: bestatic
+modules:
+  - python3-Bestatic.json
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=home
```

Comment:
Are all of these actually needed for a console application?

Reply:
Why do you need display access and home access?

Reply:
I believe I need the home access so that the user can create a website directory anywhere inside the home (as this program read and write files from the current directory). I tested with no filesystem access and it failed.  

I guess I do not really need display access. I added that so that when user clicks on localhost:8080 on terminal, a browser window opens. I thought without x11/wayland this may not work.

Reply:
Don't think that'd require display access here. That just opens the browser outside the sandbox.

Reply:
Updated the manifest.